app/config/providers.php

```diff
@@ -1,6 +1,10 @@
 <?php
 
-return [ // Ordered by ABC.
+// All names lowercase, but sentense is camelCase.
+// github must be all lowercased, but paypalSandbox should have uppercase S
+
```

Comment:
this sounds weird to keep in this file.. lets put it in the oauth tutorial markdown file